.github/workflows/awsnuke.yml

```diff
@@ -0,0 +1,122 @@
+---
+name: awsnuke
+on:
+  push:
+    paths:
+      - '.github/workflows/awsnuke.yml'
+  pull_request:
+    branches:
+      - main
+    types: [opened, edited, reopened, synchronize]
+    paths:
+      - '.github/workflows/awsnuke.yml'
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    # trigger every sunday at 12:00am
+    - cron: '0 12 * * 0'
+  workflow_dispatch:
+env:
+  AWS_REGION: "eu-west-2"
+  ENVIRONMENT_MANAGEMENT: ${{ secrets.MODERNISATION_PLATFORM_ENVIRONMENTS }}
+  TF_IN_AUTOMATION: true
+  NUKE_ACCOUNTS: ${{ secrets.MODERNISATION_PLATFORM_AUTONUKE }}
+  NUKE_BLOCKLIST_ACCOUNTS: ${{ secrets.MODERNISATION_PLATFORM_AUTONUKE_BLOCKLIST }}
+permissions:
+  id-token: write # This is required for requesting the JWT
```

Comment:
Could the write permission be moved to the job level where for jobs where it is required please? (Pleasing the scorecard security checks)

Reply:
I've moved them both to nuke job level as prereqs doesn't need them.